src/views/result.js

```diff
@@ -0,0 +1,70 @@
+
+import { quizData } from '../data.js';
+import {
+  USER_INTERFACE_ID,
+} from '../constants.js';
+
+export const  result= () => {
+// RESET THE DOM--------------------------------------
+  const userInterface = document.getElementById(USER_INTERFACE_ID);
+  userInterface.innerHTML = '';
+// ====================================
+
+// CREATION OF RESULT DOM ELEMENTS ----------------
+
+const result = document.createElement('div');
+const percentage = document.createElement('div');
+const resultNum = document.createElement('div');
+const numContainer = document.createElement('div');
+const numContainer2 = document.createElement('div');
+const score = document.createElement('img');
+const complement = document.createElement('h1');
+const complement2 = document.createElement('h1');
+const percentageNum = document.createElement('h1');
+const check = document.createElement('span');
+const restart = document.createElement('button');
+const check2 = document.createElement('span');
+// =================================================
+
+
+// ADD TEXT-CONTENT----------------------
+complement.textContent = '6';
+complement2.textContent = '4';
+percentageNum.textContent = '40%';
+check.textContent = 'check';
+check2.textContent = 'close';
+restart.textContent = 'refresh';
+// ================================
+
+
+// ADDITION OF DIFFRENT ATRRIBUTES -----------------
+score.setAttribute('class','score');
```

Comment:
Setting an attribute for class will overwrite any classes that exist on it. It is probably better to use the `add` function that you call on the `classList` to add a class

Reply:
yeah that is the right way to do it...
just fixed it...